editor/progress_dialog.cpp

```diff
@@ -166,7 +166,8 @@ void ProgressDialog::_popup() {
 
 	center_panel->set_custom_minimum_size(ms);
 
-	Window *current_window = get_last_exclusive_window();
+	Window *current_window = SceneTree::get_singleton()->get_root()->get_last_exclusive_window();
+	ERR_FAIL_NULL(current_window);
```

Comment:
Are you sure you want to distract the user using unnecessary errors? Maybe just write:

```
if(!current_window) {
	return;
}
```

Reply:
See my comment below.